language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/backend/java/inner/InterfaceClass.scala

```diff
@@ -29,6 +32,16 @@ object InterfaceClass extends StrictLogging {
         .classBuilder(interfaceName)
         .addModifiers(Modifier.FINAL, Modifier.PUBLIC)
         .addField(generateTemplateIdField(packageId, interfaceId))
+        .addFields(
+          TemplateClass
+            .generateChoicesMetadata(
+              interfaceName,
+              packagePrefixes,
+              interface.choices,
+              withPrefixes = false, // TODO: remove in #15154
```

Comment:
Removing this in the [next ticket ](https://github.com/digital-asset/daml/issues/15154)